src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -39,6 +46,27 @@ const treatmentsSchema = _.set('items.properties.treatment.properties',
     treatments.items.properties.treatment.properties
   ), treatments);
 
+const privateRecordReleasesSchema = Object.assign({}, treatments.items.properties.treatment.properties, {
+  privateMedicalRecordsReleaseAccepted: {
+    type: 'boolean'
+  },
+  'view:privateMedicalRecordsReleasePermissionRestricted': {
+    type: 'object',
+    'ui:collapsed': true,
+    properties: {}
+  },
+  treatmentCenterStreet1: {
+    type: 'string'
+  },
+  treatmentCenterStreet2: {
+    type: 'string'
+  },
+  treatmentCenterPostalCode: {
+    type: 'number'
+  },
+  treatmentCenterPhone: phone,
```

Comment:
I think this has been removed. I posted the updated schema in slack for reference. Need to update the schema in `vets-json-schema` still...